### PR TITLE
Change in GET_LOCK call to make it MariaDB compatilble

### PR DESCRIPTION
**Pull request by pavel-usov**
Change in GET_LOCK call to make it MariaDB compatilble. This should resolve Issue 5266

The fix creates a lock with a timeout of 136 years which is failry enough to be considered as "unlimited"

### CODE REVIEW

File: physical/mysql/mysql.go [hunk 1]
```diff
@@ -620,7 +620,7 @@ func (i *MySQLLock) becomeLeader() error {
 func (i *MySQLLock) Lock() error {
 	defer metrics.MeasureSince([]string{"mysql", "get_lock"}, time.Now())
 
-	rows, err := i.in.Query("SELECT GET_LOCK(?, -1), IS_USED_LOCK(?)", i.key, i.key)
+	rows, err := i.in.Query("SELECT GET_LOCK(?, 4294967295), IS_USED_LOCK(?)", i.key, i.key)
```
Comment (vishalnayak) [inline physical/mysql/mysql.go]: How was this number chosen?

Comment (pavel-usov) [reply physical/mysql/mysql.go]: This number is the decimal representation of maximal integer with 64bit length. In hexadecimal this number should be 0xffffffff

**Comment by briankassouf** [comment]
@michaeljs1990 could you take a look at this?

File: physical/mysql/mysql.go (hunk 1 above)
Comment (michaeljs1990) [reply physical/mysql/mysql.go]: Can we instead use `math.MaxUint32` from the https://golang.org/pkg/math/ package and add the comment a comment above that about the `-1` not being compatible with different MySQL flavours?

**Comment by michaeljs1990** [comment]
Thanks for the improvement to make this work on MariaDB! I admit the -1 is a rather undocumented get lock feature but it seemed common enough in other software I had worked with I didn't think it would cause issues.

File: physical/mysql/mysql.go (hunk 1 above)
Comment (pavel-usov) [reply physical/mysql/mysql.go]: @michaeljs1990 Yes, we can use this constant. I tested this, committed changes and comments.

Comment (sksingh20) [reply physical/mysql/mysql.go]: Same problem faced. Can you help with steps on how to apply this fix on a running mysql deployment?

Comment (michaeljs1990) [reply physical/mysql/mysql.go]: You shouldn't need to make any changes to mysql for this fix. What is the issue you are running into?

**Pull request merged**